include/stdexec/execution.hpp

```diff
@@ -3273,8 +3301,8 @@ namespace stdexec {
     using _Env = __1;
     using __cust_sigs =
       __types<
-        tag_invoke_t(split_t, get_completion_scheduler_t<set_value_t>(get_attrs_t(_Sender&)), _Sender),
-        tag_invoke_t(split_t, get_completion_scheduler_t<set_value_t>(get_attrs_t(_Sender&)), _Sender, _Env),
+        tag_invoke_t(split_t, get_completion_scheduler_t<set_value_t>(get_attrs_t(const _Sender&)), _Sender),
+        tag_invoke_t(split_t, get_completion_scheduler_t<set_value_t>(get_attrs_t(const _Sender&)), _Sender, _Env),
```

Comment:
question on the customizations taking env - it wasn't clear to me from the spec that split or ensure_started provide the env customized versions. can you help point that out. or are they non-standard?

Reply:
They're non-standard. I intend to propose them, though. Any algorithm that must eagerly `connect` a sender before the final receiver is available to optionally accept an environment.